ui/job-view/App.jsx

```diff
@@ -311,87 +310,83 @@ class App extends React.Component {
         <Provider store={store}>
           <Pushes filterModel={filterModel}>
             <PinnedJobs>
-              <SelectedJob>
-                <KeyboardShortcuts
+              <KeyboardShortcuts
```

Comment:
The diff looks crazy, but the only thing I did was remove the context for ``SelectedJob``, so the rest is indent.